wpengine/faustjs
# style: Style and structure the example project

Adds styling, intro content, and examples of components, global and local CSS/Sass, and locally hosted fonts for the starter example.

For [BH-827](https://wpengine.atlassian.net/browse/BH-827).

<img width="1527" alt="Screenshot 2021-02-01 at 13 40 11" src="https://user-images.githubusercontent.com/647669/106460076-11690f00-6493-11eb-83b0-997b9d412e66.png">

### To test
Follow the [Lerna setup](https://github.com/wpengine/headless-framework/blob/canary/docs/DEVELOPMENT.md#npm-packages) and update your `.env.local` file.

You should see the screenshot above when you visit the homepage.

You should see the expected content on the homepage whether Settings → Reading displays latest posts or a static page, due to https://github.com/wpengine/headless-framework/pull/109/commits/5f26f23a93ad7a9fe347e66b3a964a0a56f0cd8b.

### Notes
Menu links are currently hard-coded in Header.tsx.
